addons/sale_timesheet/views/project_task_views.xml

```diff
@@ -108,8 +108,10 @@
             </xpath>
             <xpath expr="//page[@name='settings']//field[@name='allow_billable']" position="after">
                 <div invisible="not allow_billable or not allow_timesheets" class="text-muted">
-                    Timesheets without a sales order item are
-                    <field name="billing_type" nolabel="1"/>
+                    Timesheets without a sales order item are reported as
+                    <div class="d-inline-flex">
```

Comment:
the new div should be needed since the text and field are both children of the div in 110 :thinking: 
If the text and dropdown don't fit in the same line perhaps you can reduce the width of the dropdown to ``w-75`` or ``w-50`` :shrug:

Reply:
Thank you for the suggestion @pkyriakou 🙇. I have made the required changes, please have a look 😁.